locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -49,6 +50,68 @@
 import javax.annotation.Nullable;
 import javax.xml.ws.WebServiceException;
 
+import org.apache.commons.lang3.ArrayUtils;
```

Comment:
BTW, my ordering was to keep the `org.apache.brooklyn` and `io.brooklyn` and `brooklyn` imports at the bottom, with more generic imports above. But, whatever, am happy to use the standard, have changed my IDE config.